lenguasdearagon/aragonario-frontend
# Feature/Linked words

# Description
Add links to words in reverse dictionary of entries text.

# How to test
1. Go to ```/search/?q=andar&l=es-ar```
2. Check that words in entries text that have translation ar -> es have a link to this translation
3. Test more words

# Note
You must work with this branch of aragonario: feature/linked-words (See https://github.com/lenguasdearagon/aragonario/pull/25)
